Documentation/helm-ceph-cluster.md

```diff
@@ -0,0 +1,164 @@
+|---------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------|
+| `operatorNamespace`                                     | Namespace of the Rook Operator                                                                                                                                              | `rook-ceph`             |
+| `configOverride`                                        | Cluster ceph.conf override                                                                                                                                                  | <none>                  |
+| `toolbox.enabled`                                       | Enable Ceph debugging pod deployment. See [toolbox](ceph-toolbox.md)                                                                                                        | `false`                 |
+| `toolbox.tolerations`                                   | Toolbox tolerations                                                                                                                                                         | `[]`                    |
+| `toolbox.affinity`                                      | Toolbox affinity                                                                                                                                                            | `{}`                    |
+| `dataDirHostPath`                                       | Host location to store mon and Rook data                                                                                                                                    | `/var/lib/rook`         |
+| `monitoring.enabled`                                    | Enable Prometheus integration, will also create necessary RBAC rules                                                                                                        | `false`                 |
+| `cephClusterSpec.*`                                     | Cluster configuration, see below                                                                                                                                            | See below               |
+
+### Ceph Cluster Spec
+
+The `CephCluster` CRD takes its spec from `cephClusterSpec.*`. This is not an exhaustive list of parameters.
+For the full list, see the [Cluster CRD](https://rook.github.io/docs/rook/master/ceph-cluster-crd.html).
+
+| Parameter                                               | Description                                                                                                                                                                 | Default                 |
```

Comment:
Since we can directly import all the cephcluster settings, what if we just refer to the CephCluster CR documentation? It will be much easier to maintain the doc.